<?xml version="1.0"?>
<launch>

	<arg name="rate" default="10.0" />
	<arg name="map_frame" default="map" />
	<arg name="base_link_frame" default="base_link" />
	<arg name="use_localization" default="false" />
	<arg name="desired_velocity" default="0.25" />
	
	<node name="mpc_test" pkg="kugle_mpc" type="mpc_imp_node" output="screen">
		<param name="rate" type="double" value="$(arg rate)"/>
		<param name="map_frame" type="str" value="$(arg map_frame)"/>
		<param name="base_link_frame" type="str" value="$(arg base_link_frame)"/>
		<param name="use_localization" type="bool" value="$(arg use_localization)"/>
		<param name="desired_velocity" type="double" value="$(arg desired_velocity)"/>
	</node>

</launch>
